runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -333,11 +364,23 @@ impl<T: Config> Pallet<T> {
 			// Note that `provide_multi_dispute_data` will iterate, verify, and import each
 			// dispute; so the input here must be reasonably bounded.
 			let _ = T::DisputesHandler::provide_multi_dispute_data(disputes.clone())?;
+			dispute_sets_processed_metric
+				.with_label_values(&["imported"])
+				.inc_by(disputes.len() as u64);
+
 			if T::DisputesHandler::is_frozen() {
+				// Relay chain freeze, at this point we will not include any parachain blocks.
+				dispute_sets_processed_metric.with_label_values(&["frozen"]).inc();
+
 				// The relay chain we are currently on is invalid. Proceed no further on parachains.
 				return Ok(Some(dispute_statements_weight::<T>(&disputes)).into())
 			}
 
+			// Process the dispute sets of the current session.
+			dispute_sets_processed_metric
+				.with_label_values(&["current)"])
```

Comment:
although I'm not a fan of `current`, because we accumulate for all sessions, but it's better than `current)` :)
```suggestion
				.with_label_values(&["current"])
```

Reply:
Damn, I never see them typos 🙄 . I think the rate statistic will still provide some value. I'll remove/update metrics in followup PR where I'll also implement a boilerplate for making use of these metrics.